src/main/print-downstream-assets.js

```diff
@@ -49,7 +50,7 @@ Its a new model and not present in Atlan yet, you'll see the downstream impact f
         sendSegmentEvent("dbt_ci_action_downstream_unfurl", {
             asset_guid: asset.guid,
             asset_type: asset.typeName,
-            downstream_count: downstreamAssets.length,
+            downstream_count: downstreamAssets.entities.length,
```

Comment:
If the length is greater than character limit then what should we do? Should we run the entire action again with a smaller `ASSET_LIMIT`?

Reply:
Simple strip the extra characters. content = content.slice(0, COMMENT_LIMIT);

Reply:
Would it be a good idea to strip the comment, because let's say if the user changes 5 models and the action hits the character limit after writing downstream assets for 3 of the models modified. In that case, they will be shown incomplete data, and may cause confusion.